llvm/utils/lit/tests/shtest-shell.py

```diff
@@ -4,7 +4,7 @@
 # FIXME: Temporarily dump test output so we can debug failing tests on
 # buildbots.
 # RUN: cat %t.out
-# RUN: FileCheck --input-file %t.out %s
+# RUN: FileCheck --input-file %t.out %s --dump-input-context=1000
```

Comment:
leftover from debugging?

Reply:
Yes, forgot to delete that. I will remove it in my next change.